README.md

```diff
@@ -1,5 +1,66 @@
 # phone-history-store-frontend
-
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e1c7c0497f5a487d90610ab41e9db55f)](https://app.codacy.com/gh/nezago/phone-history-store-frontend?utm_source=github.com&utm_medium=referral&utm_content=nezago/phone-history-store-frontend&utm_campaign=Badge_Grade_Dashboard)  [![Maintainability](https://api.codeclimate.com/v1/badges/e5ff93644af293ab0e46/maintainability)](https://codeclimate.com/github/nezago/phone-history-store-frontend/maintainability)  ![Node.js CI](https://github.com/nezago/phone-history-store-frontend/workflows/Node.js%20CI/badge.svg)
 
-This is the official repository of Phone History Store
+## Project vision
+Since the rise of technology has become one of the essential daily life dependency, mobile devices are one of the tools mostly used to facilitate access to the technology. Everyday we make calls to our family members, our friends and our mostly cherished lovers, we talk through phone calls, through short message service (also known as SMS).
+
+There are sometimes we find ourselves in a need of recalling our sweet memories of what we talked with our friends in the past. Hence our mobile phones, or tablets don't have enough space to save the logs, and SMS for us for a long time like 1, 2, 3 or even 10 years.
+
+Here Phone History Store helps in that situation, you can store your sms, your call logs, and you phone book as well, for any desired time. It helps you manage and save securely all your phone history, and you can get your phone history wherever you are, even if you lost your device, but you history will remain with you.
+
+## Project description
+> Description : //You describe the general logic of the project here
+
+## Screenshot (s)
+> Screenshots: // If your project has some UI, you will link the screenshot of the UI here
+
+## Built with
+> Built with : // You will specify the technologies you used to build this project such as languages, frameworks, or libararies, ...
+
+## Features
+> Features : // List of features, what a user will be able to do with this project. Eg: a user can signup
+
+## Deployments
+	* Live demo link //The link of deplyoment of your project
+	* Documentation link //Link to documentation of this project
+
+## API Endpoints Specifications
+> If this project is a backend project which exposes some APIs. List all APIs here
+	* Root API
```

Comment:
Codacy found an issue: [[no-blockquote-without-marker] Missing marker in block quote](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5868903)